app/api/ada/index.js

```diff
@@ -927,6 +947,10 @@ export default class AdaApi {
       };
 
       let unsignedTxResponse;
+      let trxMetadata = undefined;
```

Comment:
I think you can just use a ternary here so that you can make `trxMetadata` const instead